cms/static/js/views/content_group_details.js

```diff
@@ -66,31 +68,12 @@ define([
                         Translators: 'count' is number of units that the group
                         configuration is used in.
                     */
-                    'Used in %(count)s unit', 'Used in %(count)s units',
+                    'Used in {count} unit', 'Used in {count} units',
                     count
                 );
             }
 
-            return interpolate(message, { count: count }, true);
-        },
-
-        getOutlineAnchorMessage: function () {
-            var message = _.escape(gettext(
-                    /*
-                        Translators: 'outlineAnchor' is an anchor pointing to
-                        the course outline page.
-                    */
-                    'This content group is not in use. Add a content group to any unit from the %(outlineAnchor)s.'
-                )),
-                anchor = str.sprintf(
-                    '<a href="%(url)s" title="%(text)s">%(text)s</a>',
-                    {
-                            url: this.model.collection.parents[0].outlineUrl,
-                            text: _.escape(gettext('Course Outline'))
-                    }
-                );
-
-            return str.sprintf(message, {outlineAnchor: anchor});
```

Comment:
Sorry if I'm missing context, but I'm guessing this is being removed because the escaping is happening elsewhere now?